Encounter with God.html

```diff
@@ -241,7 +241,7 @@ <h3 class="heading-wbg">Dates</h3>
 										  <div class="event-prog">
                                         	<span class="timeline-stone"></span>
                                             <div class="event-prog-content">
-                                    			<strong><i class="fa fa-clock-o"></i> November 9 – 11</strong>
+                                    			<strong><i class="a fa-calendar"></i> November 9 – 11</strong>
```

Comment:
shouldn't it be class="fa fa-calendar" instead of class="a...."

Reply:
Changes has been done